frontend/packages/pipelines-plugin/src/components/pipelines/detail-page-tabs/__tests__/EventListenerDetails.spec.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import { ResourceSummary } from '@console/internal/components/utils';
+import EventListenerDetails from '../EventListenerDetails';
+import {
+  EventlistenerTestData,
+  EventlistenerTypes,
+} from '../../../../test-data/event-listener-data';
+import EventListenerURL from '../EventListenerURL';
+import EventListenerTriggers from '../EventListenerTriggers';
+
+jest.mock('react-i18next', () => {
+  const reactI18next = require.requireActual('react-i18next');
+  return {
+    ...reactI18next,
+    useTranslation: () => ({ t: (key) => key }),
+  };
+});
+
+type EventListenerDetailsProps = React.ComponentProps<typeof EventListenerDetails>;
+
+describe('EventListener Details', () => {
+  let wrapper: ShallowWrapper<EventListenerDetailsProps>;
+
+  beforeEach(() => {
+    wrapper = shallow(
+      <EventListenerDetails
+        obj={EventlistenerTestData[EventlistenerTypes.BINDINGS_TEMPLATE_NAME]}
+      />,
+    );
+  });
+  it('should render eventlistener details page', () => {
+    expect(wrapper.find(ResourceSummary).exists()).toBe(true);
+    expect(wrapper.find(EventListenerURL).exists()).toBe(true);
+    expect(wrapper.find(EventListenerTriggers).exists()).toBe(true);
+  });
```

Comment:
Careful with these kind of tests. Most of this is testing static content. It'll always be there in the current iteration of the code. So this test serves (mostly) to just test we don't change the content... which is not what tests are for.

Test just the conditionals and things that can pivot on you during prop changes. In this case, just `triggers.length` not rendering the `EventListenerTriggers` component when there are no triggers in the `eventListener`.

But this looks like you cover in your next two tests... so this test has very little value and will just serve to break when we want to change the static content.

Reply:
removed it.